packages/lab/stories/stepped-tracker/stepped-tracker.doc.stories.mdx

```diff
@@ -52,7 +52,15 @@ The SteppedTracker contains multiple TrackerStep child components. Each TrackerS
   <Story id="lab-stepped-tracker--basic" />
 </Canvas>
 
-#### Best Practice
+### Orientation
+
+The SteppedTracker can have orientation of "horizontal" or "vertical"
```

Comment:
I've asked @mikearildbrown to have a look to make sure the wording is consistent with other places

Reply:
Sounds good. Probably more important to concentrate on any changes needed for the site docs though